client/scss/layouts/_page-editor.scss

```diff
@@ -622,6 +630,154 @@ footer .preview {
   }
 }
 
+#preview-panel {
```

Comment:
We should avoid styles against an id element - this creates more specific styles that are hard to override.